cpp/benchmarks/common/generate_input.hpp

```diff
@@ -403,3 +403,37 @@ std::unique_ptr<cudf::table> create_random_table(std::vector<cudf::type_id> cons
                                                  row_count num_rows,
                                                  data_profile const& data_params = data_profile{},
                                                  unsigned seed                   = 1);
+
+/**
+ * @brief Generate sequence columns starting with value 0 in first row and increasing by 1 in
+ * subsequent rows.
+ *
+ * If the number of passed types is smaller than the number of requested columns, the types
+ * will be repeated cyclically to fill the number of requested columns.
+ *
+ * @param dtype_ids Vector of requested column types
+ * @param num_cols Number of columns in the output table
+ * @param num_rows Number of rows in the output table
+ * @param null_probability optional, probability of a null value
+   <0 implies no null mask, =0 implies all valids, >=1 implies all nulls
+ * @param seed optional, seed for the pseudo-random engine
+ * @return A table with the sequence columns.
+ */
+std::unique_ptr<cudf::table> create_sequence_table(std::vector<cudf::type_id> const& dtype_ids,
```

Comment:
Span

Reply:
span can't used with initializer list. initializer is inline, and convenient here.

Reply:
Then be explicit and make the parameter be `initializer_list`, not `vector`. Using `vector` isn't saying what you mean.

Reply:
Right. With moving `repeate_dtypes ` outside, `initializer_list` parameter can't be used in `create_sequence_table`. Also, `span` parameter can't be used with `initializer_list` argument. implicit conversion of `initializer_list` to `vector` is only way. So, leaving this as `vector`.